src/Fixer/ControlStructure/TrailingCommaInMultilineFixer.php

```diff
@@ -84,27 +85,34 @@ public function getDefinition(): FixerDefinitionInterface
 
     public function isCandidate(Tokens $tokens): bool
     {
-        return $tokens->isAnyTokenKindsFound([T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN, '(']);
+        return $tokens->isAnyTokenKindsFound([T_ARRAY, CT::T_ARRAY_SQUARE_BRACE_OPEN, '(', CT::T_DESTRUCTURING_SQUARE_BRACE_OPEN]);
     }
 
     protected function createConfigurationDefinition(): FixerConfigurationResolverInterface
     {
+        $allOptions = [
+            self::ARRAY_DESTRUCTURING,
+            self::ELEMENTS_ARGUMENTS,
+            self::ELEMENTS_ARRAYS,
+            self::ELEMENTS_PARAMETERS,
+            self::MATCH_EXPRESSIONS,
+        ];
+
         return new FixerConfigurationResolver([
             (new FixerOptionBuilder('after_heredoc', 'Whether a trailing comma should also be placed after heredoc end.'))
                 ->setAllowedTypes(['bool'])
                 ->setDefault(false)
                 ->getOption(),
-            (new FixerOptionBuilder('elements', sprintf('Where to fix multiline trailing comma (PHP >= 8.0 for `%s` and `%s`).', self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS))) // @TODO: remove text when PHP 8.0+ is required
+            (new FixerOptionBuilder('elements', sprintf('Where to fix multiline trailing comma (PHP >= 8.0 for `%s` and `%s`).', self::ELEMENTS_PARAMETERS, self::MATCH_EXPRESSIONS))) // @TODO: update text when PHP 8.0+ is required
```

Comment:
what would you see as a value when PHP 8+ is required?
I was seeing here that we could fully drop this config option.

Reply:
see answer below